docs/usecases.rst

```diff
@@ -2,57 +2,113 @@
 Use Cases
 =========
 
-ReFrame Usage at CSCS
----------------------
+ReFrame has been publicly released on May 2017, but it been used in production at the Swiss National Supercomputing Centre since December 2016.
```

Comment:
```suggestion
ReFrame has been publicly released on May 2017, but it has been used in production at the Swiss National Supercomputing Centre since December 2016.
```